src/github.com/eugenmayer/concourse-static-resource/shared/curlopt.go

```diff
@@ -1,21 +1,20 @@
 package shared
 
 import (
-	"github.com/eugenmayer/concourse-static-resource/model"
 	"fmt"
+
+	"github.com/eugenmayer/concourse-static-resource/model"
 )
```

Comment:
if yo can, remove those newlines too :)

Reply:
Another thing that was flagged: https://github.com/golang/go/wiki/CodeReviewComments#imports

Can remove if you want though?

Reply:
sure!